sample_project/Assets/StarterAssets/ThirdPersonController/Scripts/ThirdPersonController.cs

```diff
@@ -115,20 +115,23 @@ public class ThirdPersonController : MonoBehaviour
         private bool _hasAnimator;
 
         private RaycastHit hitBelow;
+        private Vector2 lastMousePosition;
 
-        private bool IsCurrentDeviceMouse
+
+        public bool IsKeyboard()
```

Comment:
This name doesn't make sense. Why are we using "IsKeyboard" to check for mouse movement?